drizzle/schema.ts

```diff
@@ -66,6 +66,7 @@ export const subscriptions = pgTable("subscriptions", {
 	trialEnd: timestamp("trial_end"),
 });
 
+type TeamType = "standard" | "internal";
```

Comment:
```suggestion
type TeamType = "customer" | "internal";
```

I'd like to propose changing the team type from `standard` to `customer`.

Reasons:
- `customer` more clearly conveys the meaning of "customer team", improving code readability
- `standard` could be confused with plan types (e.g., standard plan)
- Looking at future extensibility, naming based on organization types (customer/internal/partner) would be more consistent when we add new team types

Let me know what you think about this suggestion.

Reply:
Thank you. I think `customer` is better too 👍

Reply:
fixed [aa9c5f3](https://github.com/giselles-ai/giselle/pull/124/commits/aa9c5f3f3d7af493507a6cdf7e07b3d47cfb9e99)